Test/WhiskeyTest.psm1

```diff
@@ -1,4 +1,6 @@
 
+$PSModulesDirectoryName = 'PSModules'

```

Comment:
Should we have this variable name match the one in [Whiskey.psm1](https://github.com/webmd-health-services/Whiskey/blob/develop/Whiskey/Whiskey.psm1#L14), `$powerShellModulesDirectoryName`?

Reply:
I wanted them different so that we didn't accidentally use the global variable from the tests in Whiskey (which is what I did accidentally, and tests passed on my machine but failed on the build server; this also uncovered that Whiskey wasn't correctly failing builds in some situations.)

Reply:
Ok I thought that was your reasoning. In that case, maybe make the name more explicit that it's different for tests? `$PSModulesDirectoryNameForTests` ?